vars/buildPlugin.groovy

```diff
@@ -68,25 +68,17 @@ def call(Map params = [:]) {
                                     readFile('.mvn/extensions.xml').contains('git-changelist-maven-extension')
                             if (incrementals) { // Incrementals needs 'git status -s' to be empty at start of job
                                 if (isUnix()) {
-                                    sh(script: 'git clean -xffd > /dev/null 2>&1',
-                                       label: 'Clean for incrementals',
-                                       returnStatus: true) // Ignore failure if CLI git is not available
+                                    sh 'git clean -xffd > /dev/null 2>&1 || :' // Ignore failure if CLI git is not available
```

Comment:
If the git command is not available, this will report the 'git not found' message in the log file without failing the job.  I think that is better than not reporting the 'git not found' message and is consistent with the later use in this pull request.
```suggestion
                                    sh 'git clean -xffd || :' // Ignore failure if CLI git is not available
```